Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psd1

```diff
@@ -73,7 +73,8 @@
         "Dismount-VmfsDatastore",
         "Resize-VmfsVolume",
         "Restore-VmfsVolume",
-        "Sync-VMHostStorage"
+        "Sync-VMHostStorage",
+        "Remove-VMHostStaticiSCSITargets"
```

Comment:
Think this needs to be changed to `Remove-VMHostStaticIScsiTargets` as well

Reply:
Done